parachain/src/provider.rs

```diff
@@ -432,13 +591,14 @@ where
 		ClientType::Beefy
 	}
 
-	#[cfg(feature = "testing")]
-	async fn ibc_events(&self) -> Pin<Box<dyn Stream<Item = IbcEvent> + Send + Sync>> {
-		use futures::StreamExt;
-		use tokio_stream::wrappers::BroadcastStream;
-
-		let stream =
-			BroadcastStream::new(self.sender.subscribe()).map(|result| result.unwrap_or_default());
-		Box::pin(Box::new(stream))
+	async fn timestamp_at(&self, block_number: u64) -> Result<u64, Self::Error> {
```

Comment:
```suggestion
	async fn query_timestamp_at(&self, block_number: u64) -> Result<u64, Self::Error> {
```

I kinda like the naming convention we have going on lol